homeassistant/components/thermopro/strings.json

```diff
@@ -17,5 +17,21 @@
       "already_in_progress": "[%key:common::config_flow::abort::already_in_progress%]",
       "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
     }
+  },
+  "services": {
+    "set_datetime": {
+      "description": "Sets the Date & Time for TP358 devices",
+      "fields": {
+        "datetime": {
+          "description": "Optional DateTime with timezone to set device to",
+          "name": "Date & Time with TZ"
+        },
```

Comment:
We should use the local time of the system instead of introducing the complexity of having them being able to enter it. If their system time is wrong they have a big problem already